rules-topics/09.00_agility-abilities.md

```diff
@@ -9,254 +9,187 @@ The Agility Point cost of each ability is equal to its level.  Characters may "m
 | Level | Abilities |
 |---|---|
 | 1 | Critical Backstab |
-| 1 | Blind |
-| 1 | Slow |
-| 2 | Disarm |
-| 2 | Entangling |
-| 3 | Kneel |
+| 1 | Blinding Strike |
+| 1 | Slowing Strike |
+| 2 | Disarming Strike |
+| 2 | Entangling Strike |
+| 3 | Kneeling Strike |
 | 3 | Resourceful |
-| 4 | Parry/Spell Parry |
-| 4 | Weakness |
-| 5 | Breach |
-| 5 | Critical Silence |
+| 4 | Parry |
+| 4 | Spell Parry |
+| 4 | Weakening Strike |
+| 5 | Breaching Strike |
+| 5 | Silencing Strike |
+| 6 | Fearful Strike |
 | 6 | Waylay |
-| 6 | Fear |
-| 7 | Varible Critical |
+| 7 | Critical Empowerment |
 | 7 | Warbow Training |
-| 8 | Stun |
-| 8 | Toxin Strike |
+| 8 | Stunning Strike  |
+| 8 | Trickle Blade |
```

Comment:
Surprising Proficiency is missing here.

Reply:
Only needs to be added to the agility json